packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -153,6 +156,8 @@ skipInNode(describe)('plugin-meetings', () => {
           .then((conversation) => {
             console.log('CONVERSATION CREATED ', JSON.stringify(conversation));
 
+            this.timeout(80000);
```

Comment:
Why?

Reply:
Same reason as below, waiting for the peerConnection

Reply:
The call hasn't even been dialed, is this the right place for it?